### PR TITLE
GitHub action to run periodically or by trigger to extractor

### LINKED ISSUE #143: GitHub action to run periodically or by trigger to extract configs
For now we just need to run `./gradlew build` and commit and push to main branch all channels.

But this can be done as GitHub action, to be done daily.

Example on how to do this:
https://github.com/checkstyle/checkstyle/blob/977931a4c52791d55adce728106eef3766502e12/.github/workflows/release-maven-prepare.yml#L7-L20

https://github.com/checkstyle/checkstyle/blob/977931a4c52791d55adce728106eef3766502e12/.github/workflows/release-maven-prepare.yml#L43-L47

https://github.com/checkstyle/checkstyle/blob/977931a4c52791d55adce728106eef3766502e12/.github/workflows/release-maven-prepare.yml#L52-L54

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Resolves #143

### CODE REVIEW

**Comment by relentless-pursuit** [comment]
@romani  please review, i have fixed the review comments.

File: .github/workflows/cron-extract.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
-name: Java CI
+name: Java CI Cron Extractor
```
Comment (romani) [inline .github/workflows/cron-extract.yml:1]: Just `Cron Extractor`

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .github/workflows/cron-extract.yml (hunk 1 above)
Comment (relentless-pursuit) [reply .github/workflows/cron-extract.yml]: done

**Review by romani — APPROVED** [review]

**Pull request merged**